Automattic/jetpack
# Update jetpack extension tests to work with updated @wordpress dependencies

#### Changes proposed in this Pull Request:
* Changes to some Jetpack block test expectations to work with the newer Gutenberg @wordpress dependencies

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Check our PR and run `pnpm install`
* Run `pnpm test-extensions` in the `projects/plugins/jetpack` dir and make sure all pass
